src/registries/figure_registry.ts

```diff
@@ -58,6 +61,40 @@ function getChartMenu(
     },
     getCopyMenuItem(figureId, env),
     getCutMenuItem(figureId, env),
+    {
+      id: "copy_as_image",
+      name: _t("Copy as image"),
+      icon: "o-spreadsheet-Icon.COPY_AS_IMAGE",
```

Comment:
same as the existing `o-spreadsheet-Icon.COPY`
```suggestion
      icon: "o-spreadsheet-Icon.COPY",
```

Reply:
so twice the same icon in the same menu?

Reply:
Oh sorry, I really thought it was exactly the same icon but I wasn't checking the right menu item